templates/gcp/index.html

```diff
@@ -0,0 +1,189 @@
+{% extends "gcp/base_gcp.html" %}
+
+{% block title %}Ubuntu on GCP{% endblock %}
+{% block meta_description %}Ubuntu is the operating system platform of choice for GCP, powering innovation on public cloud in environments ranging from dev/test to mission-critical production. Run Ubuntu on Google Cloud, benefit from our worldwide mirroring and publication and talk to Canonical about how we can help optimize your cloud workloads.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1Gvt9tNFDZkqkBcP0k79C6ueTJCmNZvwoXQoU19WEa8E/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip u-no-padding--top u-no-padding--bottom">
+  <div class="p-strip--suru-topped">
+    <div class="row u-equal-height">
+      <div class="col-7 col-medium-4">
+        <h1>
+          Ubuntu on GCP
```

Comment:
This title does not match the copy doc

Reply:
So this keeps being updated, but then it makes things more inconsistent, because then there is the `Talk to us about Ubuntu on GCP` which again it's GCP instead of Google Cloud. So should I change all entries to Google Cloud? 
For now I updated it to reflect current copy doc.